cli/src/cmd/cast/run.rs

```diff
@@ -120,6 +123,11 @@ impl RunArgs {
                 pb.set_position(0);
 
                 for (index, tx) in block.transactions.into_iter().enumerate() {
+                    // arbitrum L1 transaction at the start of every block that has gas price 0
+                    // and gas limit 0 which causes reverts, so we skip it
+                    if tx.from == "0x00000000000000000000000000000000000a4b05".parse::<Address>()? {
```

Comment:
The optimism sender can be found here: https://github.com/ethereum-optimism/optimism/blob/d5ccf81966bc2fc650c802fd4b80a30e98843a0b/op-node/rollup/derive/l1_block_info.go#L26

Reply:
Didn't need to skip that because it has 1m gas limit and its enough to execute it. Had to skip arbi because their gas limit is 0 and it fails because of intrinsic gas costs > 0 and fails with OOG